### ISSUE TITLE
Replace stacks.co gradient

### ISSUE THREAD

**Issue by jasperjansz** [blockstack/ux#973]
We have a new gradient video in the works that is more like the metaverse from the rebrand. I need to replace all images, videos, and animations on site with it.

**Comment by andresgalante** [comment]
When doing this keep in mind accessibility. At the moment the white font over the bg doesn't meet AA contrast
